counsel.el

```diff
@@ -5138,6 +5141,37 @@ The optional `BUILD-ROOT' is useful for other helpers that have found
      local-files)
     compiles))
 
+(defun counsel--find-build-subdir (root)
+  "Return builds sub-directory of `ROOT', if one exists."
+  (cl-some
+   #'(lambda (x)
```

Comment:
We need to be consistent, either `#'` everywhere before `(lambda ...)` or nowhere.

Reply:
Nowhere, please. `lambda` [is self-quoting](https://www.gnu.org/software/emacs/manual/html_node/elisp/Anonymous-Functions.html).